common/men_at_arms_types/wc_cultural_maa_types.txt

```diff
@@ -0,0 +1,1689 @@
+﻿# standard costs
+@maa_buy_cost = 150
+@maa_low_maintenance_cost = 1.0
+@maa_high_maintenance_cost = 5.0
+@cultural_maa_extra_ai_score = 60 # Equivalent to having 6 extra regiments beyond what the code scoring would indicate (see NEGATIVE_SCORE_PER_EXISTING_REGIMENT)
+
+abomination = {
+	type = massive_infantry
+	
+	damage = 60
+	toughness = 80
+	pursuit = 45
+	screen = 35
+	
+	terrain_bonus = {
+		wetlands = { damage = -8 }
+		jungle = { damage = -10 }
+	}
+
+	counters = {
+		skirmishers = 1
+		light_aerial = 1
+	}
+	
+	buy_cost = { gold = 120 }
```

Comment:
So you don't use a different formula for the cultural MaAs.
That's the formula used for generic MaAs.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/125708024-01926ba1-3973-4dac-9035-fc8a27f054c2.png)

</details>

So it's: `Sums of Modifier Costs * Stack Size / 100`

Also, you should make it a bit cheaper (so it's effective to use your cultural MaAs) so Balance Factor should be about 110% or more.